### DISCUSSION TITLE
Do you Support Clickable items inside slideshow?

### DISCUSSION THREAD

**Discussion by dsievers** [Accessible360/accessible-slick#73] [Q&A]
We are trying to use your slick slideshow, but our ADA SortSite tool is giving ADA level A errors in the slideshow because you put aria-hidden=true on the container and we have anchor tags inside the slides.  Is there a setting or way to make the slides that are not visible hide the anchor tags or other tabbable elements from the screen readers?

**Comment by cramontgomery** [comment]
I think those auto-scanners have huge flaws in them. This might be one of them. The auto-scan is assuming you want people to access those links and they can't because of the aria-hidden=true.... but this is a feature, not a bug. You don't need people accessing those links unless the slide is in focus, right?
